### ISSUE TITLE
MSI add-in installer 

### ISSUE THREAD

**Issue by doronkind** [angelsix/solidworks-api#42]
@angelsix, good morning!

What do you think about to provide some examples how to create msi/exe installer for your sw add-in? I think it can be useful cause it is really lack of information about this. 

Does it works with your framework:  http://help.solidworks.com/2015/English/api/sldworksapiprogguide/Miscellaneous/Create_Setup_Project_to_Distribute_SolidWorks_Add-in.htm?id=5e152c0abbef41ad99e957ec76cb2335

I think we can use WiX Toolset, e.g. to create MSI but if you have a working example it will be great to analyze it.

**Comment by angelsix** [comment]
Yes I will do a video and code on this. I'll leave this open until I do it

**Comment by doronkind** [comment]
It would be great! Actually I am working on it now but unfortunately can't register COM object with WiX toolset. 

BTW Am I right, sw exe path is not necessary for COM registering?

**Comment by angelsix** [comment]
Correct. I just have that in my Addin Installer tool so the user clearly sees if they actually have SolidWorks installed :P

**Comment by emersonbottero** [comment]
Even in the existing an option for installing MacroFeature COM may be necessary in the future.

**Comment by doronkind** [comment]
@angelsix, do you have a plan when you approximately complete this video?

**Comment by angelsix** [comment]
Probably a month, as I go away next week then 2 weeks after that for another 2 weeks. So prob 5-6 weeks away

**Comment by doronkind** [comment]
@angelsix, it would be great. I've done MSI installer with COM registration using WiX, if you need any help I'll be glad to provide

**Comment by Venzentzheng** [comment]
Hi @doronkind I am just being in the same situation that an installer from WiX is required for my project, did you ever have permission issue that is preventing you from activating the add-in ?
